src/shared/libs/iota/utils.js

```diff
@@ -191,6 +191,36 @@ export const isValidAddress = (address) => {
     return false;
 };
 
+/**
+ * Checks if the last trit is 0
+ *
+ * @method isLastTritZero
+ * @param {string} address
+ *
+ * @returns {boolean}
+ */
+export const isLastTritZero = (address) => {
+    if (!/[E-V]/.test(address.slice(80, 81))) {
+        return true;
+    }
+    return false;
+};
+
+/**
+ * Async version of isLastTritZero
+ *
+ * @method isLastTritZeroAsync
+ * @param  {string}  address
+ *
+ * @returns {boolean}
+ */
+export const isLastTritZeroAsync = (address) => {
```

Comment:
I think adding an async util here is unnecessary. 

I can see the motivation for its addition and [usage](https://github.com/iotaledger/trinity-wallet/pull/868/files#diff-8ea5b2a5a0c274aaec60a3bbffa271c0R436) but you could also do:

```
return Promise.resolve(isLastTritZero(...)).then(() => {}).catch(() => {})
```